### PR TITLE
Dev -> Main

### PR DESCRIPTION
**Pull request by 0xHansLee**
This PR is to merge `dev` into `main` for release.

### CODE REVIEW

**Review by Pangssu — APPROVED** [review]
let's go

**Review by kangsorang — APPROVED** [review]
lvvgtm

**Review by seolaoh — APPROVED** [review]
LGTM

**Comment by 0xHansLee** [comment]
Thank you all, but let's merge this after merging #105 and #106

**Pull request merged**
